common/djangoapps/util/enterprise_helpers.py

```diff
@@ -19,7 +23,7 @@ def enterprise_enabled():
     """
     Determines whether the Enterprise app is installed
     """
-    return 'enterprise' in settings.INSTALLED_APPS
+    return 'enterprise' in settings.INSTALLED_APPS and not FAILED_ENTERPRISE_IMPORT
```

Comment:
Never mind, think I've figured this out. Enterprise wasn't in INSTALLED_APPS for the CMS, so even with the updated version installed, `enterprise.*` wasn't available in the CMS, which was causing issues because test suites for `common` items run in both LMS and CMS.